release-notes/2023-06/readme.md

```diff
@@ -0,0 +1,111 @@
+# SystemLink Enterprise release 2023-06 Release Notes
+
+The 2023-06 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## Upgrading from the release year-release-month to the release year-release-month
+
+- 'assetservice >= 0.0.283'
+    - Description: New service was introduced in SystemLink chart. Any PVC that contains assetservice in its name needs to be removed prior to the upgrade. The following privileges are being used: asset:Query, asset:Create, asset:Update, asset:Delete.
+    - Installation guide: Remove pvc that have assetservice in its name.
```

Comment:
PVCs?

Reply:
Defined it as "PersistentVolumeClaims (PVCs)" - 👍